build-logic/jvm/src/main/kotlin/build-logic.test-base.gradle.kts

```diff
@@ -21,6 +21,7 @@ tasks.configureEach<Test> {
     exclude("**/*Suite*")
     jvmArgs("-Xmx1536m")
     jvmArgs("-Djdk.net.URLClassPath.disableClassPathURLCheck=true")
+    jvmArgs("--add-opens", "java.base/java.net=ALL-UNNAMED")
```

Comment:
tests run on java 8 so this won't work

Reply:
Ok, on java8 test should work even without that flag. I will fix it later today.